json-path/src/test/java/com/jayway/jsonpath/internal/function/BaseFunctionTest.java

```diff
@@ -13,8 +13,9 @@
  */
 public class BaseFunctionTest {
     protected static final String NUMBER_SERIES = "{\"empty\": [], \"numbers\" : [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]}";
-    protected static final String TEXT_SERIES = "{\"urls\": [\"http://api.worldbank.org/countries/all/?format=json\", \"http://api.worldbank.org/countries/all/?format=json\"], \"text\" : [ \"a\", \"b\", \"c\", \"d\", \"e\", \"f\" ]}";
+    protected static final String TEXT_SERIES = "{\"urls\": [\"http://api.worldbank.org/countries/all/?format=json\", \"http://api.worldbank.org/countries/all/?format=json\"], \"text\" : [ \"a\", \"b\", \"c\", \"d\", \"d\", \"e\", \"f\" ]}";
     protected static final String TEXT_AND_NUMBER_SERIES = "{\"text\" : [ \"a\", \"b\", \"c\", \"d\", \"e\", \"f\" ], \"numbers\" : [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]}";
+    protected static final String OBJECT_SERIES = "{\"empty\": [], \"objects\" : [ {\"a\":   \"a-val\"}, {\"b\": \"b-val\"}, {   \"a\": \"a-val\"} ]}";
```

Comment:
what about array of arrays? does it work?

Reply:
Yes.
I added a test for it.